Tasks/Maven/CodeAnalysis/mavensonar.ts

```diff
@@ -15,7 +15,7 @@ import sqCommon = require('sonarqube-common/sonarqube-common');
 // Gets the SonarQube tool runner if SonarQube analysis is enabled.
 export function getSonarQubeRunner(mvnPath:string, mavenPOMFile: string, mavenOptions: string, execFileJacoco?: string): ToolRunner {
     if (!sqCommon.isSonarQubeAnalysisEnabled()) {
-        return;
+        return null;
```

Comment:
Unrelated change bundled into this PR - correctly returns null instead of just returning. This should never actually be hit as we check that SQ is enabled before calling into this method.